pyccel/parser/base.py

```diff
@@ -27,14 +27,14 @@
 
 from pyccel.ast.utilities import recognised_source
 
+from pyccel.parser.scope     import Scope
 from pyccel.parser.utilities import is_valid_filename_pyh, is_valid_filename_py
 
-from pyccel.errors.errors import Errors
+from pyccel.errors.errors   import Errors
+from pyccel.errors.messages import PYCCEL_UNFOUND_IMPORTED_MODULE
 
-# TODO - remove import * and only import what we need
-#      - use OrderedDict whenever it is possible
+# TODO - use OrderedDict whenever it is possible
```

Comment:
I agree. Should we also remove the current usage of `OrderedDict`? I would assume that `dict` would be faster

Reply:
We have compared them for usage in Psydac, and in our experiments they are equally fast. We decided to replace `OrderedDict` with `dict` because the latter can be dumped to a YAML file natively: https://github.com/pyccel/psydac/pull/157

If it doesn't cost much effort, I would also prefer using `dict` in Pyccel because it is a more basic class.

Reply:
Fixed 22e379a3